packages/map-template/src/components/BottomSheet/BottomSheet.jsx

```diff
@@ -7,12 +7,14 @@ import LocationDetails from '../LocationDetails/LocationDetails';
 import Wayfinding from '../Wayfinding/Wayfinding';
 import Directions from '../Directions/Directions';
 import Search from '../Search/Search';
+import LocationsList from '../LocationsList/LocationsList';
 
 const BOTTOM_SHEETS = {
     SEARCH: 0,
-    LOCATION_DETAILS: 1,
-    WAYFINDING: 2,
-    DIRECTIONS: 3
+    LOCATIONS_LIST: 1,
```

Comment:
I think it's fine to name the bottom sheet so it is clear that is has something to do with external ID's.
That it happens to use a more generic component to show them is just a bonus.